packages/onchainkit/src/token/types.ts

```diff
@@ -1,6 +1,7 @@
 // 🌲☀🌲
 import type { Address } from 'viem';
 import type { PortfolioTokenWithFiatValue } from '../api/types';
+import { Ref } from 'react';
```

Comment:
super nit: Definitely non blocking, but you could add `type` for consistency. We could add a ESLint rule for this if we want. Mostly thinking out loud, no need to update, unless you want to